### ISSUE TITLE
Sporadic segmentation fault with BGRA captures using DeckLink 4K Extreme 12G

### ISSUE THREAD

**Issue by dzhoshkun** [gift-surg/GIFT-Grab#62]
This problem occurred only once: https://gitlab.com/gift-surg/GIFT-Grab/-/jobs/124751443 (**not the head of the #30 branch, about to be merged into #18).
I haven't managed to reproduce it.

**Comment by dzhoshkun** [comment]
Might've been due to the absence of fbef4fc and bac3569

**Comment by joubs** [comment]
Closing as out of date.
